src/Blazored.LocalStorage/LocalStorageService.cs

```diff
@@ -248,6 +266,38 @@ public T GetItemInternal<T>(string key)
             }
         }
 
+        public object GetItemInternal(string key)

```

Comment:
```suggestion
        private object GetItemInternal(string key)
```

Is there a reason for this to be public? I'm guessing you've gone with this as the method this replaces was public but that shouldn't have been

Reply:
Also, do we need to keep the old GetItemInternal<T> method?

Reply:
I don't know, I don't remember to have touched access level.
But it is used by *RaiseOnChangingSync* (that should be renamed *RaiseOnChanging*).